src/components/WalletInit/WalletInitScreen.js

```diff
@@ -36,12 +38,40 @@ const messages = defineMessages({
     id: 'components.walletinit.walletinitscreen.restoreWalletButton',
     defaultMessage: '!!!Restore wallet',
   },
+  restoreNormalWalletLabel: {
+    id: 'components.walletinit.walletinitscreen.restoreNormalWalletLabel',
+    defaultMessage: '!!!15-word Wallet',
+  },
+  restoreNormalWalletExplanation: {
+    id: 'components.walletinit.walletinitscreen.restoreNormalWalletExplanation',
+    defaultMessage:
+      '!!!If you have a Yoroi recovery phrase consisting of 15 ' +
```

Comment:
I thought I'd keep that one for now because 1) the message is slightly different, it refers explicitly to wallets created using yoroi (which is not a requirement but may be a bit more reassuring for users) and 2) changing it would break translations. I can change it in the future to use the same generic message, once we have more translations.